layouts/404-hi.html

```diff
@@ -0,0 +1,28 @@
+{{ define "main" }}
+<div class="container-404">
+    <h4>
+        शायद आप यह देखना चाहते थे:
+    </h4>
+
+    <br />
+    <br />
+
+    <ul>
+        <li>
+            <a href="/docs">
+                कुबेरनेट्स डॉक्यूमेंटेशन
```

Comment:
`कुबरनेट्स प्रलेखन`

Reference: https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit?usp=sharing

Reply:
Okay, will change this.